Update for new GitHub Repository View

mkoch227 opened the pull request:
As indicated at https://github.com/blog/2085-a-new-look-for-repositories, a new look for respositories is coming.  Some CSS has changed for the active tab, so this PR updates the CSS to apply the necessary styles for the new CSS.

silverwind commented on github-dark.css:
```diff
@@ -658,7 +659,8 @@
   }
-  .pagehead-tabs-item.selected {
+  .pagehead-tabs-item.selected,
+  .js-selected-navigation-item.selected.reponav-item {
```
Maybe this is overqualified? Does `.reponav-item.selected` suffice?

silverwind replied:
(same applies to all additions in this PR)

silverwind commented:
Can you make the inactive tabs `#888` instead of the link color? As these are probably `a` tags, you may need to do `a.class` to outweight our `a` style.

mkoch227 commented:
I went ahead and switched the selectors to `.reponav-item.selected` as it also works, and `.js-selected-navigation-item.selected.reponav-item` did seem overqualified. I  also changed the inactive tab's text to `#888` as requested.  I've also included how it looks below in case you're unable to view the new design yet.

![screen shot 2015-11-17 at 5 46 53 pm](https://cloud.githubusercontent.com/assets/333430/11227490/43b83ace-8d54-11e5-8ff7-86b5308a99d9.png)

silverwind commented:
Thanks :+1:
